All project code/index.js

```diff
@@ -71,9 +73,12 @@ app.get('/welcome', (req, res) => {
 });
 
 app.get('/', (req, res) => {
-    res.redirect('/login'); //this will call the /login route in the API
```

Comment:
Sorry, I forgot to look over index before making this request. These two lines are functionally the same, but redirecting to login saves a bit of time.